source/PlayState.hx

```diff
@@ -1,19 +1,131 @@
 package;
 
+import flixel.FlxG;
 import flixel.FlxState;
+import flixel.group.FlxGroup.FlxTypedGroup;
+import flixel.text.FlxText;
 
 class PlayState extends FlxState
 {
+	/** Starting X coordinate of ControllableHero sprite.**/
+	static var CHXPOS:Float = 320;
+
+	/** Starting Y coordinate of ControllableHero sprite.**/
+	static var CHYPOS:Float = 300;
+
+	/** X coordinate of Haxe logo hero.**/
+	static var HEROXPOS:Float = 300;
+
+	/** Y coordinate of Haxe logo hero.**/
+	static var HEROYPOS:Float = 300;
+
+	/** X coordinate of text.**/
+	static var TEXTXPOS:Float = 235;
+
+	/** Y coordinate of text.**/
+	static var TEXTYPOS:Float = 200;
+
+	/** The displayed text.**/
+	static var TEXTTEXT:String = "Salvete, omnes!";
+
+	/** The size of the text.**/
+	static var TEXTSIZE:Int = 24;
+
+	/** The minimum starting number of ScreensaverHero sprites.**/
+	static var NUMSSHEROMIN:Int = 10;
+
+	/** The maximum starting number of ScreensaverHero sprites.**/
+	static var NUMSSHEROMAX:Int = 20;
+
+	/** The X coordinate the ScreenSaverHero sprites start.**/
+	static var SSHXPOS:Float = 320;
+
+	/** The Y coordinate the ScreenSaverHero sprites start.**/
+	static var SSHYPOS:Float = 180;
+
+	/**The size of the bounding box. This is a percentage of the play screen
+		in decimal (e.g. .75 or .5).**/
+	static var BOXSIZE:Float = .75;
+
+	/** How thick the walls of the bounding box are.**/
+	static var WALLWIDTH:Int = 5;
+
+	// Class level variables for collision detection.
+	var controllableHero:ControllableHero;
+	var screenSaverHeroGroup = new FlxTypedGroup<ScreenSaverHero>();
+	var wallGroup = new FlxTypedGroup<Wall>();
+
 	override public function create()
 	{
 		super.create();
+		controllableHero = new ControllableHero(CHXPOS, CHYPOS);
 
-		var myHero = new Hero(300, 300);
+		// Static Haxe logo Hero.
+		var myHero = new Hero(HEROXPOS, HEROYPOS);
 		add(myHero);
+
+		var text = new flixel.text.FlxText(TEXTXPOS, TEXTYPOS, 0, TEXTTEXT, TEXTSIZE);
+		text.color = 0x0080FF;
+		text.font = "Time New Roman";
+		add(text);
+
+		// Generates random number of ScreenSaverHero sprites facing and moving
+		// in different directions.
+		var numSSHeroes = FlxG.random.int(NUMSSHEROMIN, NUMSSHEROMAX);
+		for (i in 0...numSSHeroes)
+		{
+			var screenSaverHero = new ScreenSaverHero(SSHXPOS, SSHYPOS);
+			screenSaverHeroGroup.add(screenSaverHero);
+			add(screenSaverHero);
+		}
+
+		boxWall();
```

Comment:
To continue with the theme of naming conventions and self-documenting code, this method doesn't give us too clear of an idea of what it's actually doing.

It definitely involves walls, but it's unclear if they're doing something, or if they're being created, or something else.

It's good practice to make your method names start with verbs, if possible. Like `createBoxWall()` for example - it gives us a lot of clarity just by adding a single word.

It can still be good to give more context through a comment above the function (I like the one you wrote up for this down below, on lines 96-99) but it's a great thing to have our method names give people a quick preview for what the code's supposed to be doing.